lib/refile/watermark.rb

```diff
@@ -26,21 +26,20 @@ def initialize(method)
     # @param [string] gravity                  which part of the image to focus on and put the watermark on
     # @return [void]
     # @see http://www.imagemagick.org/script/command-line-options.php#gravity
-    def fill_watermark_image(img, width, height, watermark_image_filename, gravity = "Center")
+    def fill_watermark_image(img, width, height, watermark_image_filename, gravity = "Center", horizontal_margin = 0, vertical_margin = 0, opacity = 100)
```

Comment:
The default opacity in the dissolve argument below used to be 20, could you make `opacity = 20` by default instead to maintain the same default behavior?